src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -48,7 +48,6 @@ public static class ScriptConstants
         public const string LogCategoryHost = "Host";
         public const string LogCategoryFunction = "Function";
         public const string LogCategoryWorker = "Worker";
-        public const string LogCategoryMigration = "Host.Migration";
```

Comment:
There is other code to remove

Reply:
Yeah, thought about removing the properties. I'll do that.